README.md

```diff
@@ -0,0 +1,239 @@
+a student is expect to answer each one from (Such as 'True', and 'False', or 'Agree' or 'Disagree')
+
+## Feature Overview
+
+Survey and Poll are both designed to minimize the amount of fiddling a course author will have to
+do in order to create the user experience they desire. By default, answers in polls and questions in surveys
+are able to be enhanced with [Markdown](http://daringfireball.net/projects/markdown/syntax) (though it is not recommended to do more than line formatting with it)
+and images. Formatting for images is handled by the XBlock's formatters to keep a consistent and sane user experience.
+
+The feedback section of a poll or survey is shown after a user has completed the block. It, along with a poll block's
+question field, are intended to make full use of Markdown.
+
+## Installation and configuration
+
+This XBlock relies on [Xblock-utils](https://github.com/edx-solutions/xblock-utils), which should be installed first.
+
+After this, XBlock-poll may be installed using its setup.py, or if you prefer to use pip, running:
+
+    pip install /path/to/xblock_poll/repo/clone
+
+You may specify the `-e` flag if you intend to develop on the repo.
+
+### Setting up a course to use Polls and Surveys
+
+To set up a course to use the Poll and Survey XBlocks, first go to your course's outline page in the studio and look
+for the Advanced settings link in the top menus.
+
+![Advanced Settings](doc_img/advanced_settings.png)
+
+Once there, look for the *Advanced Modules List* and add `"poll"` and `"survey"` to it.
+
+![Advanced modules configuration](doc_img/advanced_modules_list.png)
+
+Save your changes, and you may now add a poll by clicking on the **Advanced Modules** button on the bottom of a
+unit editing page and selecting either `Poll` or `Survey`.
+
+## Poll Examples
+
+Polls contain a single question, letting users select from an array of answers for a choice that suits them best.
+
+Below is an example of a poll:
+
+![Poll example](doc_img/poll.png)
+
+When a user makes a selection and hits `submit`, their choice is sent to the server, and a tally of the results so far
+is presented to them.
+
+![Poll example results](doc_img/poll_result.png)
+
+The top choice's percentage is shown in *orange* while the user's selection is marked by a selected (but disabled)
+radio button on the side.
+
+### Poll variations
+
+Polls may have several customizations. Feedback may be added to a poll for display after the user has submitted their
+answer. Answers may have images associated with them. The question, feedback, and answers are all permitted to contain
+[Markdown](http://daringfireball.net/projects/markdown/syntax) (as well as arbitrary HTML) and so may be customized in the studio (as discussed in the next chapter).
+
+Here is an example of a poll that uses markdown in the question and the answers, and which uses images for each
+of the answers:
+
+![Image-only poll](doc_img/img_poll.png)
```

Comment:
I'm concerned about this from an a11y point of view. I'll let Mark comment more but want to flag this. Images-only will need to have alt text. Alternatively, we could say, these aren't accessible and you shouldn't use them or something.